columnId	description
aligned	
altitude	Altitude (m)
assembly_quality	Quality of assembly
assembly_software	Assembly software
bam_aspera	Aspera links for generated bam files. Use era-fasp or datahub name as username.
bam_bytes	size (in bytes) of generated BAM files
bam_ftp	FTP links for generated bam files
bam_galaxy	Galaxy links for generated bam files
bam_md5	MD5 checksum of generated BAM files
base_count	number of base pairs
binning_software	Binning software
bio_material	identifier for biological material including institute and collection code
bisulfite_protocol	
broad_scale_environmental_context	Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class: http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.
broker_name	broker name
cage_protocol	Link to the protocol for CAGE-seq experiments
cell_line	cell line from which the sample was obtained
cell_type	cell type from which the sample was obtained
center_name	Submitting center
checklist	ENA metadata reporting standard used to register the biosample (Checklist used)
chip_ab_provider	
chip_protocol	
chip_target	Chip target
collected_by	name of the person who collected the specimen
collection_date	Time when specimen was collected
completeness_score	Completeness score (%)
contamination_score	Contamination score (%)
control_experiment	Control experiment
country	locality of sample isolation: country names, oceans or seas, followed by regions and localities
cultivar	cultivar (cultivated variety) of plant from which sample was obtained
culture_collection	identifier for the sample culture including institute and collection code
depth	Depth (m)
dev_stage	sample obtained from an organism in a specific developmental stage
dnase_protocol	
ecotype	a population within a given species displaying traits that reflect adaptation to a local habitat
elevation	Elevation (m)
environment_biome	Environment (Biome)
environment_feature	Environment (Feature)
environment_material	Environment (Material)
environmental_medium	Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).
environmental_sample	identifies sequences derived by direct molecular isolation from an environmental DNA sample
experiment_accession	experiment accession number
experiment_alias	submitter's name for the experiment
experiment_target	
experiment_title	brief experiment title
experimental_factor	variable aspects of the experimental design
experimental_protocol	
extraction_protocol	
faang_library_selection	Library Selection for FAANG WGS/BS-Seq experiments
fastq_aspera	Aspera links for fastq files. Use era-fasp or datahub name as username.
fastq_bytes	size (in bytes) of FASTQ files
fastq_ftp	FTP links for fastq files
fastq_galaxy	Galaxy links for fastq files
fastq_md5	MD5 checksum of FASTQ files
file_location	
first_created	date when first created
first_public	date when made public
germline	the sample is an unrearranged molecule that was inherited from the parental germline
hi_c_protocol	Link to Hi-C Protocol for FAANG experiments
host	natural (as opposed to laboratory) host to the organism from which sample was obtained
host_body_site	name of body site from where the sample was obtained
host_genotype	genotype of host
host_gravidity	whether or not subject is gravid, including date due or date post-conception where applicable
host_growth_conditions	literature reference giving growth conditions of the host
host_phenotype	phenotype of host
host_sex	physical sex of the host
host_status	condition of host (eg. diseased or healthy)
host_tax_id	NCBI taxon id of the host
identified_by	name of the taxonomist who identified the specimen
instrument_model	instrument model used in sequencing experiment
instrument_platform	instrument platform used in sequencing experiment
investigation_type	the study type targeted by the sequencing
isolate	individual isolate from which sample was obtained
isolation_source	describes the physical, environmental and/or local geographical source of the sample
last_updated	date when last updated
lat	Latitude
library_construction_protocol	Library construction protocol
library_gen_protocol	
library_layout	sequencing library layout
library_max_fragment_size	
library_min_fragment_size	
library_name	sequencing library name
library_pcr_isolation_protocol	
library_prep_date	
library_prep_date_format	
library_prep_latitude	
library_prep_location	
library_prep_longitude	
library_selection	method used to select or enrich the material being sequenced
library_source	source material being sequenced
library_strategy	sequencing technique intended for the library
local_environmental_context	Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for "broad-scale environmental context". Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.
location	geographic location of isolation of the sample
location_end	
location_start	
lon	Longitude
marine_region	geographical origin of the sample as defined by the marine region
mating_type	mating type of the organism from which the sequence was obtained
ncbi_reporting_standard	NCBI metadata reporting standard used to register the biosample (Package used)
nominal_length	average fragmentation size of paired reads
nominal_sdev	standard deviation of fragmentation size of paired reads
pcr_isolation_protocol	
ph	pH
project	Project
protocol_label	the protocol used to produce the sample
read_count	number of reads
read_strand	
restriction_enzyme	
restriction_enzyme_target_sequence	The DNA sequence targeted by the restrict enzyme
restriction_site	
rna_integrity_num	
rna_prep_3_protocol	
rna_prep_5_protocol	
rna_purity_230_ratio	
rna_purity_280_ratio	
rt_prep_protocol	
run_accession	accession number
run_alias	submitter's name for the run
run_date	
salinity	Salinity (PSU)
sample_accession	sample accession number
sample_alias	submitter's name for the sample
sample_capture_status	Sample capture status
sample_collection	the method or device employed for collecting the sample
sample_description	detailed sample description
sample_material	sample material label
sample_prep_interval	
sample_prep_interval_units	
sample_storage	
sample_storage_processing	
sample_title	brief sample title
sampling_campaign	the activity within which this sample was collected
sampling_platform	the large infrastructure from which this sample was collected
sampling_site	the site/station where this sample was collection
scientific_name	scientific name
secondary_project	Secondary project
secondary_sample_accession	secondary sample accession number
secondary_study_accession	secondary study accession number
sequencing_date	
sequencing_date_format	
sequencing_location	
sequencing_longitude	
sequencing_method	sequencing method used
sequencing_primer_catalog	The catalog from which the sequencing primer library was purchased
sequencing_primer_lot	The lot identifier of the sequencing primer library
sequencing_primer_provider	The name of the company, laboratory or person that provided the sequencing primer library
serotype	serological variety of a species characterized by its antigenic properties
serovar	serological variety of a species (usually a prokaryote) characterized by its antigenic properties
sex	sex of the organism from which the sample was obtained
specimen_voucher	identifier for the sample culture including institute and collection code
sra_aspera	Aspera links for SRA data files. Use era-fasp or datahub name as username.
sra_bytes	size (in bytes) of SRA files
sra_ftp	FTP links for SRA data files
sra_galaxy	Galaxy links for SRA data files
sra_md5	MD5 checksum of atchived files
status	Status
strain	strain from which sample was obtained
study_accession	study accession number
study_alias	submitter's name for the study
study_title	brief sequencing study description
sub_species	name of sub-species of organism from which sample was obtained
sub_strain	name or identifier of a genetically or otherwise modified strain from which sample was obtained
submission_accession	submission accession number
submission_tool	Submission tool
submitted_aspera	Aspera links for submitted files. Use era-fasp or datahub name as username.
submitted_bytes	size (in bytes) of submitted files
submitted_format	format of submitted reads
submitted_ftp	FTP links for submitted files
submitted_galaxy	Galaxy links for submitted files
submitted_host_sex	physical sex of the host
submitted_md5	MD5 checksum of submitted files
submitted_read_type	submitted FASTQ read type
tag	Classification Tags
target_gene	targeted gene or locus name for marker gene studies
tax_id	NCBI taxonomic classification
taxonomic_classification	Taxonomic classification
taxonomic_identity_marker	Taxonomic identity marker
temperature	Temperature (C)
tissue_lib	tissue library from which sample was obtained
tissue_type	tissue type from which the sample was obtained
transposase_protocol	
variety	variety (varietas, a formal Linnaean rank) of organism from which sample was derived
